src/main/java/io/devlabs/keytree/domains/company/domain/UserRole.java

```diff
@@ -0,0 +1,18 @@
+package io.devlabs.keytree.domains.company.domain;
+
+public enum UserRole {
+    MASTER("MASTER"),
+    ADMIN("ROLE_ADMIN"),
+    USER("ROLE_USER")
+    ;
+
+    private final String authority;
+
+    UserRole(String authority) {
+        this.authority = authority;
+    }
+
+    public String getAuthority() {
+        return authority;
+    }
+}
```

Comment:
```suggestion
    private final String name;

    UserRole(String name) {
        this.name = name;
    }

    public String getName() {
        return name;
    }
}
```
- Authority / Role이 추후 시큐리티 도입시 차이가 있습니다. 반영할때 편하게 미리 학습하시면 좋을듯 합니다.
- 해당 코드 부분에서는 authority를 그냥 name이라는 명명 쓰시면 될것 같아요.